net/dns/resolver/tsdns_test.go

```diff
@@ -211,8 +214,12 @@ func TestRDNSNameToIPv6(t *testing.T) {
 	}
 }
 
+func newResolver(t testing.TB) *Resolver {
+	return New(t.Logf, nil /* no link monitor */, nil /* no link selector */)
```

Comment:
Should there be a test that does set up a link selector?

Reply:
Done.